extract_assets.py

```diff
@@ -111,23 +111,46 @@ def main():
     signal.signal(signal.SIGINT, SignalHandler)
 
     extractedAssetsTracker = manager.dict()
-    if os.path.exists(EXTRACTED_ASSETS_NAMEFILE) and not args.force:
+    if not args.force and os.path.exists(EXTRACTED_ASSETS_NAMEFILE):
         with open(EXTRACTED_ASSETS_NAMEFILE, encoding='utf-8') as f:
             extractedAssetsTracker.update(json.load(f, object_hook=manager.dict))
 
+    extract_text_path = "assets/text/message_data.h"
+    extract_staff_text_path = "assets/text/staff_message_data.h"
```

Comment:
I would suggest using `pathlib.Path` here, but given this script doesn't use it at all then it doesn't matter

Reply:
Yeah, there are other places that should use it (for example the ZAPD path). Feel it would be better in a different PR.